src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -14,16 +14,33 @@
   </PropertyGroup>
 
   <Target Name="NpmRunBuild" BeforeTargets="BeforeBuild">
-      <Exec Command="npm install --prefix ./ @dynamods/splash-screen@latest" />
+      <Exec Command="npm pack @dynamods/splash-screen@latest" />
```

Comment:
The first step is to download the .tgz file

Reply:
Would you add inline comments, you should be able to use `<!-- comment -->`

Reply:
Should we clean up none latest versioned npm package?